src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const discountForSixDays = 20;
+  const discountForSevenDays = 50;
+
+  if (days <= 2) {
+    return (price * days);
+  } else if (days < 7) {
+    return (price * days) - discountForSixDays;
+  } else if (days >= 7) {
+    return (price * days) - discountForSevenDays;
+  }
```

Comment:
lets create variables for days too

Reply:
also avoid such nesting `else if` conditions
This code is hard to read and maintain